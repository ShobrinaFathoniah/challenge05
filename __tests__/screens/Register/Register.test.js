import React from 'react';
import Register from '../../../src/screens/Register';
import {create} from 'react-test-renderer';

describe('Test Snapshoot', () => {
  test('Register Snapshoot', () => {
    jest.useFakeTimers();

    const snap = create(<Register />);
    expect(snap).toMatchSnapshot();
  });
});
