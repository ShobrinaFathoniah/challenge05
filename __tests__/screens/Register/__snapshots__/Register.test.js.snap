// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Snapshoot Register Snapshoot 1`] = `
<View>
  <Text>
    Register
  </Text>
</View>
`;
